### DISCUSSION TITLE
Recommend some good English quotes

### DISCUSSION THREAD

**Discussion by zztqsh** [zztqsh/THEO1126.github.io#5] [Q&A] [answered]
Recommend some good English quotes  ✏️

**Comment by zztqsh** [comment]
Recommend some good English quotes

**Comment by THEO1126** [comment, marked as answer]
The world may be full of cheating, however wenever lack friends with a warm heart.

**Comment by THEO1126** [comment]
The rational and the irrational complement each other. Individually, they are far less powerful.
